src/test/pmemblk_create/README

```diff
@@ -0,0 +1,11 @@
+Linux NVM Library
+
+This is src/test/pmemblk_create/README.
+
+This directory contains a unit test for pmemblk_create().
```

Comment:
Your updated test program is puzzling.  You create the pmemblk pool but then instead of using libpmemblk to read or write the file, you map it again and use write(2) on the file!  So your test trashes the pool header and never actually does anything to determine if a blk pool was created.  I think you should, at a minimum, call the _check routine on the pool as I suggested.

Your statement that all errors return the same thing is incorrect.  Permission denied returns EACCES, an incorrect argument detected by the library returns EINVAL, etc.  This is a unit test, you have access to the log messages printed by LOG().  If you are testing a specific code path, use a .match file to determine if that path was taken as expected.  That's why the unittest framework captures all that information.